easybuild/easyconfigs/t/TRIQS_cthyb/TRIQS_cthyb-2.2.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS_cthyb'
```

Comment:
@rokzitko I hate to be pedantic, but do you have a compelling reason to use `TRIQS_cthyb` rather than `TRIQS-cthyb`?
I'm in favor of the latter, the underscore looks a bit weird to me in a software name...

I did a quick count: we currently have only 24 software packages with `_` in the name (not including these `TRIQS_*` ones), vs 173 with `-`.